src/commands/synthetics/__tests__/reporters/default.test.ts

```diff
@@ -32,4 +32,101 @@ describe('Default reporter', () => {
       writeMock.mockClear()
     }
   })
+
+  describe('testTrigger', () => {
+    beforeEach(() => {
+      writeMock.mockClear()
+    })
+
+    const testObject: Pick<Test, 'name'> = {
+      name: 'Request on example.org',
+    }
+    const testId = 'aaa-bbb-ccc'
+
+    const cases: [string, ExecutionRule, ConfigOverride, RegExp][] = [
+      [
+        'Blocking test, without config overwrite',
+        ExecutionRule.BLOCKING,
+        {},
+        /\[.*aaa-bbb-ccc.*\].*Found test.*Request on example\.org.*\n?$/,
```

Comment:
I had to rely on regular expressions here as the output contains ANSI characters included by the Chalk library.

We could use the [strip-ansi](https://github.com/chalk/strip-ansi) library to remove these characters, and test the text in a stricter manner.

I didn't want to import another library in this project, but it's doable. Feel free to suggest it if you think it would be better 🙂

Reply:
Using Regex here doesn't bother me personally, or we could also use `chalk.` in the expected output maybe